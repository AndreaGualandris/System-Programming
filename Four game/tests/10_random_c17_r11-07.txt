7
13
0
8
2
6
15
8
6
11
12
13
11
16
1
12
15
6
4
7
3
16
14
4
4
12
5
9
1
6
8
10
8
15
12
11
14
11
6
14
6
8
14
4
16
12
9
0
13
13
2
12
13
5
7
13
2
7
8
2
6
5
3
13
9
11
7
9
6
3
15
16
3
0
0
0
15
6
13
5
7
9
2
4
0
10
2
12
12
7
0
11
7
9
2
0
12
12
13
3
7
0
1
6
16
5
2
6
7
14
3
14
2
12
6
11
2
5
14
10
1
1
3
13
1
11
16
2
4
13
12
7
15
12
2
7
14
12
13
5
16
4
13
4
14
11
11
11
6
12
8
4
5
14
8
10
4
5
4
11
5
16
2
7
5
8
5
6
16
3
15
1
12
1
10
5
4
16
15
11
10
5
12
1
11
12
0
