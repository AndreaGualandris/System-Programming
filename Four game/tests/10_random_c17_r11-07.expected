-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
-----------------
01234567890123456
Player 'o' to move:
       o
       v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
       o         
-----------------
01234567890123456
Player 'x' to move:
             x
             v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
       o     x   
-----------------
01234567890123456
Player 'o' to move:
o
v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
o      o     x   
-----------------
01234567890123456
Player 'x' to move:
        x
        v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
o      ox    x   
-----------------
01234567890123456
Player 'o' to move:
  o
  v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
o o    ox    x   
-----------------
01234567890123456
Player 'x' to move:
      x
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
o o   xox    x   
-----------------
01234567890123456
Player 'o' to move:
               o
               v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
o o   xox    x o 
-----------------
01234567890123456
Player 'x' to move:
        x
        v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
        x        
o o   xox    x o 
-----------------
01234567890123456
Player 'o' to move:
      o
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
      o x        
o o   xox    x o 
-----------------
01234567890123456
Player 'x' to move:
           x
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
      o x        
o o   xox  x x o 
-----------------
01234567890123456
Player 'o' to move:
            o
            v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
      o x        
o o   xox  xox o 
-----------------
01234567890123456
Player 'x' to move:
             x
             v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
      o x    x   
o o   xox  xox o 
-----------------
01234567890123456
Player 'o' to move:
           o
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
      o x  o x   
o o   xox  xox o 
-----------------
01234567890123456
Player 'x' to move:
                x
                v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
      o x  o x   
o o   xox  xox ox
-----------------
01234567890123456
Player 'o' to move:
 o
 v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
      o x  o x   
ooo   xox  xox ox
-----------------
01234567890123456
Player 'x' to move:
            x
            v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
      o x  oxx   
ooo   xox  xox ox
-----------------
01234567890123456
Player 'o' to move:
               o
               v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
      o x  oxx o 
ooo   xox  xox ox
-----------------
01234567890123456
Player 'x' to move:
      x
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
      x          
      o x  oxx o 
ooo   xox  xox ox
-----------------
01234567890123456
Player 'o' to move:
    o
    v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
      x          
      o x  oxx o 
ooo o xox  xox ox
-----------------
01234567890123456
Player 'x' to move:
       x
       v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
      x          
      oxx  oxx o 
ooo o xox  xox ox
-----------------
01234567890123456
Player 'o' to move:
   o
   v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
      x          
      oxx  oxx o 
ooooo xox  xox ox
-----------------
01234567890123456
Player 'o' wins!
